index.html

```diff
@@ -48,186 +57,220 @@ <h3><a href="#">LET'S CONNECT</a></h3>
+                </div>
+                <div class="work-topic">
+                    <h2>Uber Navigation</h2>
+                </div>
+                <div class="work-details">
+                    <ul>
+                        <li>Uber</li>
+                        <li class="work-2nd">∘ Lead Developer</li>
+                        <li class="work-2nd">∘ 2015</li>
+                    </ul>
+                </div>
+                <div class="work-paragraph">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-langs">
+                     <ul>
+                    <li><a href="#">html</a></li>
+                    <li class="ruby"><a href="#">Ruby on rails</a></li>
+                    <li><a href="#">css</a></li>
+                    <li><a href="#">javascript</a></li>
+                     </ul>
+                </div>
+                <div class="work-button">
+                    <button>See Project</button>
+                </div>
+            </div>
+            <div class="work-section4">
+                <img src="image-resources/Snapshoot Portfolio3.jpg" alt="">
+            </div>
+        </section>
     </section>
 
-    <section class="about">
-        <div class="about-heading">
-            <h2>About Myself</h2>
-        </div>
-        <div class="about-description">
-            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        </div>
-        <div class="about-connect">
-            <h3><a href="">LET’S CONNECT</a></h3>
+    <section class="about" id="about">
+        <div class="langs-about">
+            <div class="about-heading">
+                <h2>About Myself</h2>
+            </div>
+            <div class="about-description">
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="about-connect">
+                <h3><a href="">LET’S CONNECT</a></h3>
+            </div>
+            <div class="about-socials">
+                <div class="social-bar">
+                    <ul>
+                        <li><a href="https://twitter.com/Abbas_sDev" class="connect"><img src="image-resources/twitter.png" alt="Twitter logo"></a></li>
+                        <li><a href="https://www.linkedin.com/in/abbas-sarwar-4a0b16257/" class="connect"><img src="image-resources/linkedin.png" alt="LinkedIn logo"></a></li>
+                        <li><a href="https://github.com/AbbasSarwar" class="connect"><img src="image-resources/Github.png" alt="Github logo"></a></li>
```

Comment:
- [ ] I think it would be better to adjust the styling of the About section by:
    - Increasing the ```margin-top``` to give more space between the work section and the about section as given on the Figma design 👍 
    - Adjusting the padding of your about section also to replicate the Figma design you may also reduce the width of each of you language, 🙏 

![image](https://user-images.githubusercontent.com/102757126/220879235-cde44143-9fef-4849-8c88-8251a91a73c3.png)